mrp_product_variants/__init__.py

```diff
@@ -18,3 +17,18 @@
 ##############################################################################
 
 from . import models
+from openerp import SUPERUSER_ID, api
+
+
+def assign_product_template(cr, registry):
+    """
+    This post-init-hook will update all existing mrp.bom.line
+    """
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+    for line in env['mrp.bom.line'].search([]):
```

Comment:
These operations can be better done with SQL for performance reasons installing the module